src/Home/index.tsx

```diff
@@ -1,10 +1,143 @@
-import { Center, Box } from 'native-base';
-import React from 'react';
+import React, { useState } from 'react';
+import { FontAwesome } from '@expo/vector-icons';
+import {
+  AddIcon,
+  Box,
+  Circle,
+  Divider,
+  HStack,
+  SearchIcon,
+  useToast,
+  VStack,
+} from 'native-base';
+import { ScrollView } from 'react-native';
+import { Tile } from './tile';
+import { ModalSelection } from './ModalSelection';
+import { API_URL } from '../constants';
+import { Game } from '../interface';
+
+export function Home({ navigation, player }: any) {
+  const [isLoading, setIsLoading] = useState<boolean>(true);
```

Comment:
Why do you set a `isLoading` state if you don't use it to render a loader or to prevent double-clicking on a buttton?

Reply:
I added it because I was going to use it to display a loader. Which I do in the next PR. (I shouldn't have added it so early)

Reply:
Ok, let's keep it